Lab_16_Virtual_Pet_pt1/src/VirtualPet.ts

```diff
@@ -0,0 +1,40 @@
+export default class VirtualPet {
+    name: string 
+    hunger: number
+    happiness: number
+
+    constructor(name: string, hunger: number = 50, happiness: number = 50) {
+        this.name = name
+        this.hunger = hunger
+        this.happiness = happiness
+    }
+
+    describe() {
```

Comment:
Class method return values can be annotated too:

```suggestion
    describe(): string {
```